types/enzyme/tslint.json

```diff
@@ -3,6 +3,7 @@
     "rules": {
         "dt-header": false,
         "no-duplicate-imports": false,
-        "no-unnecessary-generics": false
+        "no-unnecessary-generics": false,
+        "no-redundant-jsdoc-2": false
```

Comment:
See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46006#issuecomment-658956797

Reply:
Do update the lint file. merge with master and things should work out. @orta is fixing this kind of issue.

Reply:
Do you mean _don't_ update the lint config? i.e. I should revert this, merge master, and this will disappear?

Reply:
@sheetalkamat Could you clarify?

Reply:
I'm going to try reverting it and seeing if it's working now.